app/models/private_message_topic_tracking_state.rb

```diff
@@ -185,6 +188,17 @@ def self.publish_user_archived(topic, user_id)
     MessageBus.publish(self.user_channel(user_id), message, user_ids: [user_id])
   end
 
+  def self.publish_read(topic_id, last_read_post_number, user, notification_level = nil)
```

Comment:
How come we can't just call `publish_read_message` directly instead of having `publish_read` call it? (ditto for the other state class)

Reply:
This is to avoid leaking the `message_type` and `channel` concerns into `TopicUser`

Reply:
Oh yes I see now, thanks